bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -192,6 +196,18 @@ spec:
             - name: CASSANDRA_ENABLE_USER_DEFINED_FUNCTIONS
               value: {{ .Values.cluster.enableUDF | quote }}
             {{- end }}
+            {{- if kindIs "valid" .Values.host.ports.cassandra.intraPort }}
+            - name: CASSANDRA_TRANSPORT_PORT_NUMBER
+              value: {{ .Values.host.ports.cassandra.intraPort | quote }}
+            {{- end }}
+            {{- if kindIs "valid" .Values.host.ports.cassandra.jmxPort }}
+            - name: CASSANDRA_JMX_PORT_NUMBER
+              value: {{ .Values.host.ports.cassandra.jmxPort | quote }}
+            {{- end }}
+            {{- if kindIs "valid" .Values.host.ports.cassandra.cqlPort }}
+            - name: CASSANDRA_CQL_PORT_NUMBER
+              value: {{ .Values.host.ports.cassandra.cqlPort | quote }}
+            {{- end }}
```

Comment:
As I mentioned before, this should be based on a `containerPorts` array parameter (for consistency with other charts).